docs/overview/Glossary.md

```diff
@@ -236,6 +236,9 @@ The _manifest_ is a subset of the [election description](#election-description)
 
 ### master nonce
 
+!!! failure "Deprecated"
+    this term is deprecated and replaced with [mediator](#mediator)
```

Comment:
As far as I could tell these two terms were identical.  I marked this as deprecated because it's not used anywhere else, but please correct if I'm wrong.